engine/src/main/java/org/terasology/logic/ai/AICommands.java

```diff
@@ -25,26 +25,38 @@
 import com.google.common.collect.Iterables;
 
 /**
+ * Artificial Intelligence command
  */
 @RegisterSystem
 public class AICommands extends BaseComponentSystem {
     @In
     private EntityManager entityManager;
 
+    /**
+     * Lists all AIs in the world
+     * @return String string containing number of simple AIs and hierarchical AIs
+     */
     @Command(runOnServer = true, shortDescription = "Count all AIs in the world")
     public String countAI() {
         int simpleAIs = Iterables.size(entityManager.getEntitiesWith(SimpleAIComponent.class));
         int hierarchical = Iterables.size(entityManager.getEntitiesWith(HierarchicalAIComponent.class));
         return "Simple AIs: " + simpleAIs + ", Hierarchical AIs: " + hierarchical;
     }
 
+    /**
+     * Destroys all AIs in the world
```

Comment:
This needs to include what is meant by destroying AIs. In this case the command goes through all *entities* with AI components attached, then destroys those entities. The return doc is unchanged from the former method, which is bad since it does differ, in this case in returns counts for how many were *destroyed*